.github/workflows/test-amazon-web-services-install.yaml

```diff
@@ -77,6 +77,7 @@ jobs:
           --nodes 3
 
     - name: Install PostHog using the Helm chart
+      id: helm_install
```

Comment:
non-blocking

Is this actually used at all?

Reply:
Not yet but I find useful to keep the steps identical across e2e tests so maybe we will be able to extract and move them out to reduce copy paste (not a pri for now)